main.js

```diff
@@ -16,26 +16,54 @@ const removeClass = (el, oldClass) => {
   const inputField = document.getElementsByTagName('input')[0];
   let typeTimeout;
   const TYPE_INTERVAL = 10000;
+  const siteContainer = document.getElementsByTagName('body');
 
   inputField.addEventListener("input", (event) => {
     const value = event.target.value;
     if (typeTimeout) window.clearTimeout(typeTimeout);
 
     if (value) {
       typeTimeout = delay(matchKeyword(value), TYPE_INTERVAL);
+    } else {
+      removeClass(siteContainer[0], 'filtering');
+      
+      const prevFilter = document.getElementsByClassName('visible');
+    
+      oldEmojis(prevFilter);
     }
   }, false);
 
   const matchKeyword = (value) => {
     let matchedEmojiTitles = [];
     const matchedObjects = EMOJIS.filter(filterByKeyword.bind(this, value));
+
+
     if (matchedObjects.length == 0) return;
 
     matchedObjects.map((obj) => {
       matchedEmojiTitles.push(obj.keywords[0]);
     });
 
-    // do something with matchedEmojiTitles here
+    const prevFilter = document.getElementsByClassName('visible');
+
+    oldEmojis(prevFilter);
+
+
+    for (const emoji of matchedEmojiTitles) {
+      const emojiItem = document.querySelector('[title="' + emoji + '"]');
+
+      if (emojiItem) {
+        addClass(emojiItem.parentElement,'visible');
+      }
+    }
```

Comment:
Good idea! 

I'm looking into this right now, and for some reason even setting it as a string isn't working. Like when I use the exact code you have above and then console.log() query I just get `[title="${emoji}"]`

... weird, but I'm going to look into it!